webview/package.json

```diff
@@ -47,7 +47,6 @@
     "@storybook/manager-webpack5": "6.5.9",
     "@storybook/preset-scss": "1.0.3",
     "@storybook/react": "6.5.9",
-    "@storybook/testing-library": "0.0.13",
```

Comment:
I think you'll need to pin the other version (maybe just remove that one as it is out of order)